utils/file.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import (
+	"cronicle/ui/constants"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type WriteParams struct {
+	Message, Date, Tags string
+}
+
+type UpdateParams struct {
+	Message, Date, Tags string
+	Number              int
+}
+
+func WriteToFile(m []string, t string) {
```

Comment:
Can we rewrite this so it's more generic? It should simply take a filepath and some contents to write to that filepath.